File: docs/versioned_docs/version-1.11.0/API-Reference/python-examples/api-reference-api-examples/get-version.py
import os

import requests

url = f"{os.getenv('LANGFLOW_SERVER_URL', '')}/api/v1/version"

headers = {
    "accept": "application/json",
}

response = requests.request("GET", url, headers=headers)
response.raise_for_status()

print(response.text)

File: docs/versioned_docs/version-1.12.0/API-Reference/javascript-examples/api-reference-api-examples/get-version.js
const url = `${process.env.LANGFLOW_SERVER_URL ?? ""}/api/v1/version`;

const options = {
  method: 'GET',
  headers: {
    "accept": `application/json`,
  },
};

fetch(url, options)
  .then(async (response) => {
    if (!response.ok) {
      throw new Error(`HTTP ${response.status}`);
    }
    const text = await response.text();
    console.log(text);
  })
  .catch((error) => console.error(error));
